### PR TITLE
update

### PR DESCRIPTION
**Pull request by Yolanda18-star**
@langyuemeng @YiniXu9506 @you06 PTAL

### CODE REVIEW

File: take-you-through-the-isolation-level-of-tidb-1.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+<div class="caption-center">图1 - 可序列化和不可序列化的对比
+</div>
+
+## ANSI SQL-92
+
+ANSI SQL-92 提出了最经典的隔离级别定义，包括读未提交（Read Uncommitted）、读提交（Read Committed）、可重复读（Repeatable Read）和可序列化（Serializable）。
+
+
+|    | Dirty Read  | ANon-repeatable Read  | Phantom Read  |
```
Comment (you06) [inline take-you-through-the-isolation-level-of-tidb-1.md:41]:
```suggestion
|    | Dirty Read  | Non-repeatable Read  | Phantom Read  |
```

File: take-you-through-the-isolation-level-of-tidb-1.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+Phantom Read 是 Non-repeatable Read 的 predicate 的版本，这两种异常现象都需要在一个事务中进行两次读操作，Non-repeatable Read 指的是两次 item 类型的读操作读到了不同的内容，而 Phantom Read 则是指两次 predicate 类型的读操作读到了不同的结果。如图 2 所示，左图表示的是 Non-repeatable Read，右图表示的是 Phantom Read。
+
+![2](media/take-you-through-the-isolation-level-of-tidb-1/2.png)
+
+<div class="caption-center">图2 - Non-repeatable Read 与 Phantom Read</div>
+
+![例1](media/take-you-through-the-isolation-level-of-tidb-1/例1.png)
+
+<div class="caption-center">例1 - 虚假的 Phantom Read</div>
```
Comment (you06) [inline take-you-through-the-isolation-level-of-tidb-1.md:70]: The cases can be shown as code blocks eg.

```sql
/* Txn1 */ select * from accounts; -- 0 rows
/* Txn2 */ insert into accounts values(“tidb”, 100);
/* Txn2 */ commit;
/* Txn1 */ insert into accounts values(“tidb”, 1000); -- duplicate entry “tidb”
/* Txn1 */ select * from accounts; -- 0 rows
/* Txn1 */ select * from accounts for update; -- 1 rows
/* Txn1 */ commit;
```

**Review by you06 — APPROVED** [review]
LGTM

**Comment by langyuemeng** [comment]
LGTM

**Pull request merged**